setup.py

```diff
@@ -58,6 +58,8 @@ def get_version(package):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
```

Comment:
Any reason to not also support Python 3.4 ? (Or 3.3?)
Should there also be a `'Programming Language :: Python :: 3',` ?

Reply:
I've not done the test on Python 3.4 or older so I'm not sure if anything can go wrong. I think we can stay with Python v3.5 and v3.6 for now. Once I finish the test suite for the SDK, we can add support for other versions.